test/verify/check-system-info

```diff
@@ -421,6 +425,65 @@ class TestSystemInfo(MachineCase):
 
         self.machine.execute("umount /sys/class/dmi")
 
+    def testCPUSecurityMitigations(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go('/system/hwinfo')
+        b.enter_page('/system/hwinfo')
+        b.wait_present('#hwinfo a:contains(Mitigations)')
+        b.click('#hwinfo a:contains(Mitigations)')
+        b.wait_present('#cpu-mitigations-dialog tr:contains(No mitigations available)')
+
+        m.write('/usr/local/bin/lscpu', lscpu)
+        m.execute('chmod +x /usr/local/bin/lscpu')
+        b.reload(ignore_cache=True)
+        b.enter_page('/system/hwinfo')
+        b.wait_present('#hwinfo a:contains(Mitigations)')
+        b.click('#hwinfo a:contains(Mitigations)')
+        b.wait_present('#cpu-mitigations-dialog tr:contains(nosmt)')
+        b.wait_in_text('#cpu-mitigations-dialog tr:contains(nosmt) .btn.active', 'Off')
```

Comment:
These code blocks all look very similar. Please introduce a helper function that e. g. accepts an `lscpu` input (number of threads) and a `cmdline` value (possibly None), and `expect_link_enabled`, `expect_smt_state` arguments, etc.